backend/registration-service/.drone.yml

```diff
@@ -0,0 +1,61 @@
+kind: pipeline
+type: kubernetes
+name: build-test
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+
+  - name: docker
+    image: plugins/docker
+    settings:
+      dockerfile: backend/registration-service/Dockerfile
+      registry: quay.io
+      context: backend/registration-service
+      tags:
+      - latest
```

Comment:
Note that this will likely change later, we probably don't want to be pushing to latest, or at the very least we want to push to a tag as well, whatever that might be.